packages/klevu-ui/src/components/klevu-product/klevu-product.css

```diff
@@ -229,3 +229,9 @@ klevu-button.addToCart {
     background-position: 0% 50%;
   }
 }
+
+.ratings {
+  display: flex;
+  gap: 5px;
```

Comment:
Instead of hardcoded gap value we should use `var(--klevu-spacing-x)` to give people ability to change spacing easily.

Reply:
replaced